### PR TITLE
Created p.md

### PR DESCRIPTION
**Pull request by shyamtawli**
### Description
Issue #1566 
``<p>`` tag file created.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/html/concepts/elements/terms/p/p.md [hunk 1]
```diff
@@ -11,19 +11,16 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-Paragraph element `<p>` contains and displays a block of text. They are among the most common text-based HTML elements.
+The **`<p>`** element, also known as the paragraph element, contains and displays a block of text. They are among the most common text-based HTML elements.
 
-They are block-level, which means it is as wide as it's parent element and as tall as its content's height.
+They are block-level, which means it is as wide as it's parent element and as tall as its content's height. In addition, browsers automatically add a single blank line before and after earch <p> element.
```
Comment (KTom101) [inline content/html/concepts/elements/terms/p/p.md:16]:
```suggestion
They are block-level, which means it is as wide as its parent element and as tall as its content's height. In addition, browsers automatically add a single blank line before and after each <p> element.
```

File: content/html/concepts/elements/terms/p/p.md [hunk 2]
```diff
@@ -11,19 +11,16 @@ CatalogContent:
...
 
 ## Syntax
 
 The pargraph element is enclosed by an opening `<p>` tag and a closing `</p>` tag:
```
Comment (KTom101) [inline content/html/concepts/elements/terms/p/p.md:20]:
```suggestion
The paragraph element is enclosed by an opening `<p>` tag and a closing `</p>` tag:
```

File: content/html/concepts/elements/terms/p/p.md [hunk 3]
```diff
@@ -11,19 +11,16 @@ CatalogContent:
...
 
 ```html
```
Comment (KTom101) [inline content/html/concepts/elements/terms/p/p.md:22]:
```suggestion
```pseudo
```

**Review by KTom101 — COMMENTED** [review]
Added some changes.

**Review by KTom101 — APPROVED** [review]
LGTM 👍

**Comment by KTom101** [comment]
@yangc95 Was there any additional changes you wanted to add before merging?

**Comment by KTom101** [comment]
@shyamtawli The live changes are [here](https://www.codecademy.com/resources/docs/html/elements/p). I'm noticing you're not listed as a contributor: 

Screen Shot 2023-02-11 at 11 15 49 PM

If you wish to be added as a contributor you have to add your GitHub username to your Codecademy profile (instructions below). If you don't want to be listed, there is nothing else you have to do. 

To update your profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

**Comment by shyamtawli** [comment]
Thanks @KTom101, I have added my username on Profile.

**Pull request merged**